mitodl/ocw-studio
# Added management command to ensure single source of truth for example config files

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #141 

#### What's this PR do?
Adds a command to generate equivalent JSON example configs for the frontend based on the YML config files in the localdev directory (i.e.: the single source of truth)

#### How should this be manually tested?
- Run `manage.py override_site_config`. It should work as it did before, and should take params to target a different config/starter. 
- Run `manage.py generate_example_configs`. This should take every yml file in `localdev/configs` and create an equivalent JSON file in `static/js/resources`
